### ISSUE TITLE
Get tweets no longer works

### ISSUE THREAD

**Issue by philipperemy** [mahrtayyab/tweety#46]
```
app.get_tweets(user_id)
```
no longer works. Seems like it happens after Elon Musk posted some limitations.

**Comment by mahrtayyab** [comment]
Yes even getting tweets is behind the authentication , the library will be updated today

**Comment by philipperemy** [comment]
@mahrtayyab thank you very much 👍 👍

**Comment by bengabp** [comment]
> Yes even getting tweets is behind the authentication , the library will be updated today

@mahrtayyab please checkout my pull request, I've added 2 more features and one RateLimitError class which is a custom Exception

**Comment by bengabp** [comment]
> @mahrtayyab thank you very much 👍 👍 

@philipperemy you could use authenticated cookies at the moment, just create a dummy twitter account and copy the cookies.

**Comment by philipperemy** [comment]
@bengabp I'm not sure I get it for the cookies.

**Comment by bengabp** [comment]
@philipperemy
- ## Get the cookies while logged in to twitter

![image](https://github.com/mahrtayyab/tweety/assets/84660669/2de4b351-b04f-46a8-8e48-606d28641e64)

- ## Copy the curl command and paste [here](https://curlconverter.com/python/)

![image](https://github.com/mahrtayyab/tweety/assets/84660669/28410e39-8e7e-407b-b426-3501bd0759b9)

- ## Copy your cookies

![image](https://github.com/mahrtayyab/tweety/assets/84660669/3a8a3912-2014-48c0-aef9-b4baeb076efa)


- ## Create twitter object
```python
twitter = Twitter(cookies=cookies) #replace cookies with your cookies
```

**Comment by philipperemy** [comment]
@bengabp gotcha. Is that necessary for it to work?

**Comment by bengabp** [comment]
yes

**Comment by mahrtayyab** [comment]
It is fixed , do consider upgrading
